packages/ui/src/card-button/CardButton.tsx

```diff
@@ -109,6 +111,20 @@ const useStyles = makeStyles<
       marginRight: theme.spacing(-0.5),
     },
 
+    errorIcon: {
+      marginRight: theme.spacing(1),
+      marginLeft: theme.spacing(-0.5),
+      '& svg': {
+        fill: Color.Red300,
+      },
```

Comment:
@umrzoq-toshkentov Can we set any icon as an error icon? Will `fill` work with any of them properly?

Reply:
yes we can set any icon. That is why I just targeted to its svg, which means svg has both material ui component and our custom icons as well.

Reply:
@umrzoq-toshkentov How [alerts](https://ui.superdispatch.org/?path=/docs/lab-alert--basic) are implemented? I just think maybe we do not need to set color for icon here.
If we want a default icon for the error state, maybe we can add it directly into the component with the ability to replace it or disable it?

Reply:
Removed errorIcon and unnecessary classNames

Reply:
@umrzoq-toshkentov Does the design team know that there will not be an icon by default?

Reply:
@timurhamitov Yes I have replied it in new-color-palette threads, you can check it